homatransport/src/common/AppmasterBase.ned

```diff
@@ -44,24 +44,15 @@ module HostBase extends NodeBase
         // no of UDP apps. Specify the app types in INI file with
         // udpApp[0..1].typename="UDPVideoStreamCli" syntax
         int numUdpApps = default(0);
-        bool hasTransportScheme = default(numTrafficGeneratorApp>0);
-        bool hasTcp = default(numTcpApps>0);
-        bool hasUdp = default((numUdpApps>0) || (numTrafficGeneratorApp>0));
-
-        // tcp implementation (e.g. ~TCP, ~TCP_lwIP, ~TCP_NSC) or ~TCPSpoof
-        string tcpType =
-            default(firstAvailable("TCP", "TCP_lwIP", "TCP_NSC", "TCP_None"));
-        string udpType = default(firstAvailable("UDP","UDP_None"));
-        string transportSchemeType =
-                default(firstAvailable("HomaTransport", "TransportSchemeNone"));
-        forwarding = default(false);  // disable routing by default
-        networkLayer.proxyARP = default(false);
 
         double edgeLinkDelay @unit(us);
         double fabricLinkDelay @unit(us);
```

Comment:
Should these be a property of AppMaster? I think we should move them to the NicBase.

Reply:
These were used to pass to the workload generator module (which is part of the appmaster), so that it could compute the ideal message delays, etc.